samples/client/petstore/cpp-ue4/Private/OpenAPIPetApiOperations.cpp

```diff
@@ -220,7 +220,6 @@ void OpenAPIPetApi::FindPetsByStatusResponse::SetHttpResponseCode(EHttpResponseC
 	switch ((int)InHttpResponseCode)
 	{
 	case 200:
-	default:
```

Comment:
Has something else changed? These default clauses are needed if I remember correctly.

Reply:
Yes, there was a fix. Previously the first response code is considered the `default`, which is incorrect as `default` should be defined separately.

Shall we revise the code to throw an exception instead as the response code is not defined in the spec?

Reply:
No, this code does not support exceptions. Essentially if the response code was undefined in the spec I assumed it was a correct response. You may change that default behavior but I believe my tests ended up concluding that this was the best case scenario.

Reply:
What about logging an error message instead?

Reply:
I think I'll merge this PR which aims to fix the string comparison. 

The issue (default case has gone) is present in the generator regardless of this PR.